tekton/verify-tekton-pipeline-cosign-signated-images/resource.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: signed-pod
+spec:
+  containers:
+    - name: signed-container
+      image: index.docker.io/arunvel1988/signedimage:latest  # Assumes this is a signed image
```

Comment:
For sample policies, please use the Kyverno test images:

- ghcr.io/kyverno/test-verify-image:signed (see https://kyverno.io/policies/other/verify-image/verify-image/)


Its best to avoid dependencies on other repositories and registries.